R/print.R

```diff
@@ -4,11 +4,18 @@
 #' @param x The result object.
 #' @param ... Ignored.
 #' @return The JSON result.
-#' 
+#'
 #' @importFrom jsonlite prettify toJSON
 #' @export
 #' @method print gh_response
 
 print.gh_response <- function(x, ...) {
-  print(toJSON(unclass(x), pretty = TRUE, auto_unbox = TRUE))
+  if("raw" %in% class(x)) {
+    attr(x, c("method")) <- NULL
+    attr(x, c("response")) <- NULL
+    attr(x, ".send_headers") <- NULL
```

Comment:
Maybe we just want `attributes(x) <- NULL`?

Reply:
If we do this we lose the class, and I thought it would be useful that it still prints the the class is c("gh_response", "raw"), to remind the user of the origin and that they can query the attributes.